unit-tests/src/testing_utils/neutron_adapters/neutron_module.rs

```diff
@@ -0,0 +1,282 @@
+use cosmwasm_std::{
+    coin, to_json_binary, to_json_string, Addr, Api, BlockInfo, CustomMsg, CustomQuery, StdError,
+    StdResult, Storage,
+};
```

Comment:
We already have it in some other repo right? 
In this case I think it is better to export it to its own repo, and reuse the same thing, might make some more work because you do stuff you don't need, but will make this module much better and complete instead of having 2 copies, we would have a more complete 1

Reply:
I think it's gonna have a fair bit of new additions so I figured it's easier to manage here than coordinate updates in the other one

Reply:
Yea, but what we are doing here, is we are mocking the custom functionality of neutron module, it is unrelated to the specific use of orbital or covenant.
So no matter what you will actually add here, it should work for no matter what contract is using it, theres nothing specific about this to a specific contract.

Separating this to its own repo, will make this module more complete, the module should not change based on how a specific contract is interacting with it, rather the module should do its own thing correctly so we can see the contract is correct.
Probably not do it now, but it would save time for the future.